html/settings/_settings.scss

```diff
@@ -1059,6 +1059,71 @@ $sprk-checkbox-custom-input-cursor-disabled: default !default;
 // The background color for the custom checkbox input when disabled.
 $sprk-checkbox-custom-input-background-color-disabled: transparent !default;
 
+//
+// Huge Checkbox
+///
+/// The max-width of the huge checkbox input.
+$sprk-checkbox-input-huge-max-width: $sprk-input-container-huge-max-width !default;
+/// The transition of the huge checkbox input.
+$sprk-checkbox-huge-container-transition: $sprk-checkbox-transition !default;
+/// The border settings of the huge checkbox input.
+$sprk-checkbox-huge-container-border: 2px $sprk-black-tint-50 solid !default;
+/// The border-radius of the huge checkbox input.
+$sprk-checkbox-huge-container-border-radius: 4px !default;
+/// The padding of the huge checkbox input.
+$sprk-checkbox-huge-container-padding: 20px 16px 20px 55px !default;
```

Comment:
Lets use our spacing vars here where we can, can use space-m in place of that 16px